pkg/predict/predictor.go

```diff
@@ -197,30 +198,49 @@ func (p *Predictor) GetSchema() (*openapi3.T, error) {
 	return openapi3.NewLoader().LoadFromData(body)
 }
 
-func buildInputValidationErrorMessage(errorResponse *ValidationErrorResponse) error {
+func (p *Predictor) endpoint() string {
+	if p.isTrain {
+		return "trainings"
+	}
+	return "predictions"
+}
+
+func (p *Predictor) url() string {
+	return fmt.Sprintf("http://localhost:%d/%s", p.port, p.endpoint())
+}
+
+func (p *Predictor) buildInputValidationErrorMessage(errorResponse *ValidationErrorResponse) error {
 	errorMessages := []string{}
 
 	for _, validationError := range errorResponse.Detail {
 		if len(validationError.Location) != 3 || validationError.Location[0] != "body" || validationError.Location[1] != "input" {
 			responseBody, _ := json.MarshalIndent(errorResponse, "", "\t")
-			return fmt.Errorf("/predictions call returned status 422, and there was an unexpected message in response:\n\n%s", responseBody)
+			return fmt.Errorf("/%s call returned status 422, and there was an unexpected message in response:\n\n%s", p.endpoint(), responseBody)
 		}
 
 		errorMessages = append(errorMessages, fmt.Sprintf("- %s: %s", validationError.Location[2], validationError.Message))
 	}
 
+	command := "predict"
+	if p.isTrain {
+		command = "train"
+	}
+
 	return fmt.Errorf(
-		`The inputs you passed to cog predict could not be validated:
+		`The inputs you passed to cog %s could not be validated:
```

Comment:
nit: using the `%[1]s` [explicit argument index syntax](https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes) would cut down on repeated positional args and the likelihood of confusion that comes with them

Reply:
Resolved in [0050e8e](https://github.com/replicate/cog/pull/2013/commits/0050e8e6ab2b926ef0a8860e71e675a9d4de4ef6)